### PR TITLE
fix 1971/11/15 missing setlist #52

### PR DESCRIPTION
**Pull request by jacobcharleskillian**
fix 1971/11/15 missing setlist #52

### CODE REVIEW

**Comment by jacobcharleskillian** [comment]
Thanks again Jeff!

File: data/gdshowsdb/1971.yaml [hunk 1]
```diff
@@ -1330,7 +1330,85 @@
...
+    - :uuid: 8144cd03-560a-4ef1-879e-070dadb5ced7
+      :name: Beat It On Down The Line
+      :segued: false
+    - :uuid: 0ead60da-6abf-4589-98dd-e56faca0c767
+      :name: Dark Star
+      :segued: true
+    - :uuid: c77efe2c-7f2f-4168-acb6-18d0f40913cb
+      :name: El Paso
+      :segued: true
+    - :uuid: 0ead60da-6abf-4589-98dd-e56faca0c767
```
Comment (jefmsmit) [inline data/gdshowsdb/1971.yaml]: Deadlists has this as "Space" and Archive as "Jam". How about Jam for this one?

**Comment by jacobcharleskillian** [comment]
1971 - "Jam" works.
1991 - "Space".

File: data/gdshowsdb/1971.yaml (hunk 1 above)
Comment (jacobcharleskillian) [reply data/gdshowsdb/1971.yaml]: Listened to it today.  Hints of Dark Star in the jam, calling it Dark Star Jam

    - :uuid: 0ead60da-6abf-4589-98dd-e56faca0c767
      :name: Dark Star
      :segued: true
    - :uuid: c77efe2c-7f2f-4168-acb6-18d0f40913cb
      :name: El Paso
      :segued: true
    - :uuid: 28138053-4eda-4f61-9e75-95f3a1341ffc
      :name: Dark Star Jam
      :segued: true

**Pull request merged**